doc/Type/List.pod6

```diff
@@ -175,6 +175,69 @@ If C<$topic> is I<not> an L<Iterable|/type/Iterable>, returns the invocant if
 the invocant has no elements or its first element is a L<Match|/type/Match>
 object (this behavior powers C<m:g//> smartmatch), or C<False> otherwise.
 
+=head2 routine list
+
+Defined as:
+
+    multi sub    list(+list)
+    multi method list(List:D:)
+
+The method just returns the invocant L<self|/routine/self>. The subroutine
+adheres to the L<single argument rule|/language/functions#Slurpy_conventions>:
+if called with a single argument that is a non-L<itemized|/language/mop#VAR>
+C<Iterable> it returns a C<List> based on the argument's
+L<iterator|/routine/iterator>; otherwise it just returns the argument list.
+
+For example:
+
+    my $tuple = (1, 2);         # an itemized List
+    put $tuple.list.raku;       # OUTPUT: «(1, 2)␤»
+    put list($tuple).raku;      # OUTPUT: «($(1, 2),)␤»
+    put list(|$tuple).raku;     # OUTPUT: «(1, 2)␤»
+
+The last statement uses the L«C<prefix:<|>>|/language/operators#prefix_|»
+operator to flatten the tuple into an argument list, so it is equivalent to:
+
+    put list(1, 2).raku;        # OUTPUT: «(1, 2)␤»
+
+There are other ways to list the elements of an itemized single argument. For
+example, you can L«decontainerize|/routine/<>» the argument or use the L<C<@>
+list contextualizer|/type/Any#index-entry-@_list_contextualizer>:
+
+    put list($tuple<>).raku;    # OUTPUT: «(1, 2)␤»
+    put list(@$tuple).raku;     # OUTPUT: «(1, 2)␤»
+
+Note that listing the elements of a type object may not do what you expect:
```

Comment:
Maybe just use "casting to a list" here?

Reply:
In Raku we "coerce" rather than "cast", don't we? But coercion typically results in a specific type, so how about "converting a type object to a list"?

Reply:
Whatever that's not "listing" is probably OK. Your call.